source/usage-examples/findOneAndDelete.txt

```diff
@@ -0,0 +1,81 @@
+===================
+Delete One Document
+===================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can delete a single document in a collection with
+``collection.findOneAndDelete()``.
+The ``findOneAndDelete()`` method uses a query document that you provide
+to match only the subset of the documents in the collection that match
+the query. If you don't provide a query document (or if you provide an
+empty document), MongoDB matches all documents in the collection.
+However, only the first matched document is deleted.
+
+You can also define additional query options such as
+`sort <https://docs.mongodb.com/manual/reference/bson-type-comparison-order/index.html>`_
+and
+`projection <https://docs.mongodb.com/manual/tutorial/project-fields-from-query-results/>`_
+to configure the returned document. You should specify these options
```

Comment:
"You should" doesn't feel like the right construction here.

Reply:
Can you suggest a superior construction? It sounds fine to me.

Reply:
Changed to "you can" for now, but that doesn't seem like much of a difference to me. Would appreciate some help on this.

Reply:
I would just put "Specify these options..." but that's just me.

Reply:
Done!